content/source/docs/enterprise-beta/api/policies.html.md

```diff
@@ -0,0 +1,256 @@
+---
+layout: enterprise2
+page_title: "Policies - API Docs - Terraform Enterprise Beta"
+sidebar_current: "docs-enterprise2-api-policy"
+---
+
+# Policies API
+
+-> **Note**: These API endpoints are in beta and may be subject to change.
+
+[Sentinel Policy as Code](/docs/enterprise-beta/sentinel/index.html) is an embedded policy as code framework integrated with Terraform Enterprise. Policies are defined in organizations and enforced on all workspace Runs between a plan and an apply. This doc covers operations to create, read, update, and delete the Sentinel Policies in an organization. The [Runs API](/docs/enterprise-beta/api/run.html) covers the steps for reading and overriding policy checks in a workspace run.
+
+
+## Create a Policy
+
+This endpoint enables you to create a policy and associate it with an organization.
+
+| Method | Path           |
+| :----- | :------------- |
+| POST | /organizations/:organization/policies |
+
+### Parameters
+
+- `:organization` (`string: <required>`) - specifies the organization name where the Policy will be defined
+- `name` (`string: <required>`) - specifies the name of the policy. Accepts alphanumeric characters, as well as `-` and `_`. Cannot be modified after creation.
+- `mode` (`string: "soft"`) - specifies the enforcement level of the policy, it can be `hard`, `soft`, or `advisory`.
```

Comment:
Hard policies will block the plan from proceeding to the apply step. Soft policies will block the plan from proceeding but can be overridden by an organization owner. Advisory policies will just be a warning after the plan and does not block any action.

Reply:
we could either add this here or link to the manage-policies section

Reply:
Thinking we should just link it so we have a source of truth.